acceptancetests/jujupy/client.py

```diff
@@ -2429,9 +2429,26 @@ def _wait_for_status(self, reporter, translate, exc_type=StatusNotMet,
             reporter.finish()
         return status
 
+    def wait_for_application(self, application, timeout=60):
+        """Wait till config returns without error. This is used as a
```

Comment:
Technically not pep257 :-), should be single line summary then multiline description. i.e.

"""wait untill config returns without error

This is used....
"""